savant/deepstream/meta/frame.py

```diff
@@ -48,6 +52,16 @@ def objects_number(self) -> int:
         """
         return self.frame_meta.num_obj_meta
 
+    @property
+    def tags(self) -> Dict[str, Any]:
+        """Returns tags of frame. These tags are part of the meta information about
+        the frame that comes with the frames in the module.
+
+        :return: Dictionary with tags
+        """
+        source_meta = METADATA_STORAGE[self.source_id]
+        return source_meta.by_idx.get(self.savant_frame_meta.idx).tags
```

Comment:
Did you try this with uridecodebin source? savant_frame_meta can be empty. Please, use the helper function metadata_get_frame_meta

Reply:
I used metadata_get_frame_meta to get the GSTFrameMeta